packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.html

```diff
@@ -11,19 +11,26 @@
       class="collectionSelector mb-2 d-flex align-items-center"
       *ngIf="collections != null"
     >
-      <span>Choose a collection: </span>
-      <div class="eventSelector">
-        <label for="event">Event</label>
-        <select
-          id="collection"
-          name="collection"
-          (change)="changeCollection($event.target.value)"
-        >
-          <option value="" selected disabled hidden>Choose Collection</option>
-          <option *ngFor="let collection of collections" [value]="collection">
-            {{ collection }}
-          </option>
-        </select>
+      <div class="d-flex align-items-center">
+        <span>Choose a collection: </span>
+        <div class="eventSelector">
+          <label for="event">Event</label>
+          <select
+            id="collection"
+            name="collection"
+            (change)="changeCollection($event.target.value)"
+          >
+            <option value="" selected disabled hidden>Choose Collection</option>
+            <option *ngFor="let collection of collections" [value]="collection">
+              {{ collection }}
+            </option>
+          </select>
+        </div>
+      </div>
+      <div class="checkbox-wrapper" *ngIf="showingCollection">
+        <mat-checkbox (change)="toggleInvisible($event.checked)">
```

Comment:
No need for a wrapper here. Once you use flex the right way, the item will become right aligned.